packages/web-components/src/components/ic-button/ic-button.css

```diff
@@ -326,7 +326,7 @@ div.icon-container {
   align-items: center;
 }
 
-::slotted(svg) {
+::slotted(*) {
```

Comment:
Isn't the MUI icon wrapped in an SVG element? Might be worth displaying the MUI and button variant in different ways (in the React storybook - then we can copy this across into the guidance site as an example):
- Parent div with the slot and nested MUI icon
- slot on the MUI icon (if possible)
- using innerHTML on a div (I think this will output the same way as the first bullet point)

..and then check if we an use a more specific selector for `slotted`. Can this also be done for side navigation please.

Reply:
I found a way to import the Material Icons yesterday (see image below). I've tried importing MUI icons in our `SlottedSVG` component in past and I've struggled. I believe this is because they are [Material Design Icons (MDI)](https://pictogrammers.com/docs/library/mdi/getting-started/react/) exported as React components.

MDI, however, seem to be SVGs. Material Design has a similar component to our `SlottedSVG` component called `Icon` component and so MDI can be passed through our `SlottedSVG` component in the `path` prop in the same way as you would with their `Icon` component.

<img width="608" alt="Screenshot 2023-02-16 at 14 09 40" src="https://user-images.githubusercontent.com/113986285/219390185-6d087e3f-9ade-48f0-a19a-cb7a40ebbf30.png">

Reply:
Thank you! This is helpful for testing on a test app! I'm not sure if there's a way to do this in our story examples though? Other than if we installed mdi/js or mui icons

Reply:
> I found a way to import the Material Icons yesterday (see image below). I've tried importing MUI icons in our `SlottedSVG` component in past and I've struggled. I believe this is because they are [Material Design Icons (MDI)](https://pictogrammers.com/docs/library/mdi/getting-started/react/) exported as React components.
> 
> MDI, however, seem to be SVGs. Material Design has a similar component to our `SlottedSVG` component called `Icon` component and so MDI can be passed through our `SlottedSVG` component in the `path` prop in the same way as you would with their `Icon` component.
> 
> <img alt="Screenshot 2023-02-16 at 14 09 40" width="608" src="https://user-images.githubusercontent.com/113986285/219390185-6d087e3f-9ade-48f0-a19a-cb7a40ebbf30.png">

So using the MUI Icon does return the SVG element hence why it doesn't work with our `slottedSVG`. `SlottedSVG` works by using the path or whatever elements are nested within the svg such as `<circle>`. Probably got an error because it was rendering 
```
<svg slot='foo'>
   <svg></svg> // MUI Icon
</svg>
```
I've created a ticket to add more guidance on this. https://github.com/mi6/ic-design-system/issues/301